sympy/combinatorics/fp_groups.py

```diff
@@ -277,6 +282,50 @@ def __str__(self):
     __repr__ = __str__
 
 
+class FpSubgroup(DefaultPrinting):
+    '''
+    The class implementing a subgroup of an FpGroup (only finite index subgroups are
+    supported at this point). This is to be used if one wishes to check if an element
+    of the original group belongs to the subgroup
+
+    '''
+    def __init__(self, G, gens):
+        super(FpSubgroup,self).__init__()
+        self.parent = G
+        self.generators = list(set([g for g in gens if g != G.identity]))
+        self.C = None
+
+    def __contains__(self, g):
+        if not g in self.parent:
+            raise ValueError
```

Comment:
Is this better than returning `False`? (BTW, Python prefers the form `g not in`. More readable, I guess.)